server/routes/participant-user.js

```diff
@@ -106,7 +106,7 @@ router.patch(
       });
       res.status(200).json(updateResults);
     } else {
-      res.status(422).send(`No expression of interest with for user: ${userId}`);
+      res.status(422).send(`No expression of interest with for user`);
```

Comment:
Grammar: "No expression of interest found for user"

Reply:
Actually, should we keep it consistent with _'No expression of interest found for this participant'_?

Reply:
Yeah, that's better.